packages/main/src/plugin/util/exec.spec.ts

```
@@ -26,8 +26,9 @@ import * as sudo from '@expo/sudo-prompt';
 import type { Mock } from 'vitest';
 import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest';
 
-import * as util from '../../util.js';
-import type { Proxy } from '../proxy.js';
+import type { Proxy } from '/@/plugin/proxy.js';
```

Biome (2.3.14)

[error] Do not shadow the global "Proxy" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)